docs/integrator/recipe-development.md

```
@@ -166,7 +166,7 @@ componentRefs:
         version: "580.82.07"
 ```
 
-**Kustomize components:**
+#### Kustomize components
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)